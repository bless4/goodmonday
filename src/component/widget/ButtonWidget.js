import React from 'react';
import { Button } from 'reactstrap';

export default class ButtonWidget extends React.Component {
    render() {
        const { props } = this;
        return(
            <Button color="success" onClick={props.onClick}>Get Result</Button>
        );
    }
}