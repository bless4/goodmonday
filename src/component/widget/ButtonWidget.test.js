import React from 'react';
import { shallow } from 'enzyme';
import ButtonWidget from './ButtonWidget';

describe('ButtonWidget', () => {
  let component = null;
  
  it('renders correctly', () => {
    component = shallow(<ButtonWidget />);
  });

  it('ButtonWidget snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
