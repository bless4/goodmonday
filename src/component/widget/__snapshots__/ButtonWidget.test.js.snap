// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonWidget ButtonWidget snapshot 1`] = `ShallowWrapper {}`;
